README.md

```diff
@@ -0,0 +1,37 @@
+# fernuni-bot a.k.a. Boty McBotface 
+*Bot für den Discord Server "FU Hagen Informatik/Mathematik"*
+
+## Funktion
+### April 2020
+✨*Geburtstunde*  
+**Ursprünglich**: Rollenzuschreibung / -entfernung auf dem Server (Studiengangszugehörigkeit).
+
+### Mai 2021
+**Rolle**: Freund und Helfer bei der Organisation des Studiums und der gelegentlichen Prokrastination 🤖  
+
+* Begrüßung von neuen Servermitgliedern
+  * Direktnachricht
+  * Text in #welcome (WIP - [Issue zum Thema](https://github.com/FU-Hagen-Discord/fernuni-bot/issues/27))
+* Rollenzuschreibung / -entfernung
+  * Studiengangszugehörigkeit
+  * [Newsrolle](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/master/news_cog.py) -> Ping bei Neuigkeiten der Fak. für Informatik und Mathematik
+  * Farben für Usernames
+  * *Sonstige Rollen bei saisonalen Events*
+* Wissensmanagement
+  * Modulinformationen : [PR zur Funktion](https://github.com/FU-Hagen-Discord/fernuni-bot/pull/3)
+  * Sammlung Nützlicher [Links](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/master/links_cog.py) in den Channels
+* [Lerngruppenverwaltung](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/master/learninggroups.py)
+  * Anfragen/ Erstellen/ Umbenennen/ Löschen
+  * Ownership -> Toggle: 🔴vollzählig/ 🌲offen für neue Mitglieder
+* [Kalenderfunktion](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/master/appointments_cog.py)
+* [Umfragefunktion](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/master/poll_cog.py)
+* Text-commands
+  * Motivations- / Prokrastinationssprüche
+  * Statistiken des Servers anzeigen
+  * [Arminsagt-Sprüche](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/master/armin.py)
+* (Saisonal) Event-Funktionen
+
+## Contributing guide</h2>
+Tipp: [Issues](https://github.com/FU-Hagen-Discord/fernuni-bot/issues) anschauen und
+* eine nicht-attribuierte Issue, die den Label **good first issue** trägt, bearbeiten.
+* Bedürfnis entdeckt, der noch keine Issue hat? Issue erstellen und auf Rückmeldung warten 👍
```

Comment:
Was hältst du von sowas wie: "Dir fehlt bei Boty eine Funktion? Dann erstell eine Issue und warte auf Rückmeldung" oder in Kürze: "Dann gib `!idee <Deine Idee>` auf dem Server ein." 
Evtl. solltest du auch noch erwähnen, dass man sich auch an der Entwicklung beteiligen kann

Reply:
Habe es jetzt aktualisiert mit dem Hinweis auf !idee in dem Channel oder auf offene Issues ✔️